salt/edx/hacks.sls

```diff
@@ -45,47 +33,10 @@ copy_select_static_assets_to_static_subfolder:
     - preserve: True
 {% endif %}
 
-{% if 'tumbleweed' != salt.grains.get('edx_codename') %}
-set_from_bulk_email_address_in_lms_production_file:
-  file.append:
-    - name: /edx/app/edxapp/edx-platform/lms/envs/production.py
-    - text: EMAIL_USE_DEFAULT_FROM_FOR_BULK = ENV_TOKENS.get('EMAIL_USE_DEFAULT_FROM_FOR_BULK', False)
-
-{% for app in ['lms', 'cms'] %}
-add_sentry_integration_to_{{ app }}_production_file:
-  file.append:
-    - name: /edx/app/edxapp/edx-platform/{{ app }}/envs/production.py
-    - text: RAVEN_CONFIG = ENV_TOKENS.get('RAVEN_CONFIG', {})
-{% endfor %}
-
 {% if 'mitxpro' in salt.grains.get('environment') %}
 add_social_auth_https_redirect_to_lms_production_file:
   file.append:
     - name: /edx/app/edxapp/edx-platform/lms/envs/production.py
     - text: SOCIAL_AUTH_REDIRECT_IS_HTTPS = ENV_TOKENS.get('SOCIAL_AUTH_REDIRECT_IS_HTTPS', True)
```

Comment:
We should add this directly to edx-platform along with the xpro_base_url rather than relying on appending to it at build time.

Reply:
This looks like it's defined/used in multiple places in the configuration repo and set as `false`